app/src/main/java/org/fossasia/susi/ai/Utils/UIutils.kt

```diff
@@ -0,0 +1,23 @@
+package org.fossasia.susi.ai.Utils
+
+import android.widget.ImageView
+import com.squareup.picasso.Picasso
+import org.fossasia.susi.ai.R
+import org.fossasia.susi.ai.rest.clients.BaseUrl
+import org.fossasia.susi.ai.rest.responses.susi.SkillData
+
+class UIutils {
```

Comment:
There must be a utils file already, use that

Reply:
There is no utils file there is once mediaUtilfile for microphone and audio handling and networkUtil for network checking

Reply:
Why is the package name in capitals?

Also, change the class to object and name to `Utils` 

Besides `UIutils` is not valid camel case

Reply:
There's already a helper package, add the class there